vulcanai2/models/basenetwork.py

```diff
@@ -217,44 +247,61 @@ def hook(module, input, output):
         
         return summary
     
-    def get_layers(self):
-        return self._modules
-
-    def get_weights(self):
+    # def get_layers(self):
+    #     # TODO: priya define.
+    #     # TODO: this param doesn't actually exist
```

Comment:
This is actually PyTorch nn.Module parameter. Ref: https://pytorch.org/docs/stable/_modules/torch/nn/modules/module.html

Reply:
oh, so it's because it's subclassing nn.Module? ok doing now. Sorry and thanks